internal/controller/state/dataplane/configuration.go

```diff
@@ -1064,6 +1064,7 @@ func buildBaseHTTPConfig(
 		// HTTP2 should be enabled by default
 		HTTP2:                   true,
 		IPFamily:                Dual,
+		Policies:                buildPolicies(gateway, gateway.Policies),
```

Comment:
Is this going to build client settings policies objects too and attach to this struct?